packages/app/context/theme.tsx

```diff
@@ -1,12 +1,16 @@
-import { createContext, useReducer } from 'react';
+import { createContext, useEffect, useReducer, useState } from 'react';
 import { theme, darkTheme } from '../theme';
 import ThirdPartyThemeProviders from './ThirdPartyThemeProviders';
 import React from 'react';
+import { Platform } from 'react-native';
+import AsyncStorage from '@react-native-async-storage/async-storage';
```

Comment:
We have storage util to save data on the storage, can't we  just use that for storing data?

Reply:
Ok, I will use that. that definitely seems a better option.